Tests are failing on Travis Build (Spring Boot Admin Server)

Currently, the `master' branch build fails on travis, module **Spring Boot Admin Server**:
See https://travis-ci.org/github/codecentric/spring-boot-admin/builds/748830892

```
...
[ERROR] Failures: 
[ERROR]   InstancesProxyControllerIntegrationTest.setUpClient:41->AbstractInstancesProxyControllerIntegrationTest.setUpClient:98->AbstractInstancesProxyControllerIntegrationTest.registerInstance:229 VerifySubscriber timed out on checkpoint("Body from GET http://localhost:34145/instances/events [DefaultClientResponse]")
[ERROR]   InstancesProxyControllerIntegrationTest>AbstractInstancesProxyControllerIntegrationTest.should_forward_requests_to_mutliple_instances:161->AbstractInstancesProxyControllerIntegrationTest.registerInstance:229 VerifySubscriber timed out on checkpoint("Body from GET http://localhost:33047/instances/events [DefaultClientResponse]")
[ERROR] Errors: 
[ERROR]   InstancesProxyControllerIntegrationTest.setUpClient:46->AbstractInstancesProxyControllerIntegrationTest.setUpClient:98->AbstractInstancesProxyControllerIntegrationTest.registerInstance:225->AbstractInstancesProxyControllerIntegrationTest.getEventStream:257 » IllegalState
...
```

The Travis Build running with PR #1557 reports a test failure in `de.codecentric.boot.admin.SpringBootAdminHazelcastApplicationTest`:
https://travis-ci.org/github/codecentric/spring-boot-admin/builds/749315838
```java
java.lang.IllegalStateException: Failed to load ApplicationContext
Caused by: org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'notificationTrigger' defined in class path resource [de/codecentric/boot/admin/server/config/AdminServerHazelcastAutoConfiguration$NotifierTriggerConfiguration.class]: Bean instantiation via factory method failed; nested exception is org.springframework.beans.BeanInstantiationException: Failed to instantiate [de.codecentric.boot.admin.server.notify.NotificationTrigger]: Factory method 'notificationTrigger' threw exception; nested exception is com.hazelcast.config.InvalidConfigurationException: IMap eviction config doesn't support max size policy `ENTRY_COUNT`. Please select a valid one: [PER_NODE, PER_PARTITION, USED_HEAP_PERCENTAGE, USED_HEAP_SIZE, FREE_HEAP_PERCENTAGE, FREE_HEAP_SIZE, USED_NATIVE_MEMORY_SIZE, USED_NATIVE_MEMORY_PERCENTAGE, FREE_NATIVE_MEMORY_SIZE, FREE_NATIVE_MEMORY_PERCENTAGE].
Caused by: org.springframework.beans.BeanInstantiationException: Failed to instantiate [de.codecentric.boot.admin.server.notify.NotificationTrigger]: Factory method 'notificationTrigger' threw exception; nested exception is com.hazelcast.config.InvalidConfigurationException: IMap eviction config doesn't support max size policy `ENTRY_COUNT`. Please select a valid one: [PER_NODE, PER_PARTITION, USED_HEAP_PERCENTAGE, USED_HEAP_SIZE, FREE_HEAP_PERCENTAGE, FREE_HEAP_SIZE, USED_NATIVE_MEMORY_SIZE, USED_NATIVE_MEMORY_PERCENTAGE, FREE_NATIVE_MEMORY_SIZE, FREE_NATIVE_MEMORY_PERCENTAGE].
Caused by: com.hazelcast.config.InvalidConfigurationException: IMap eviction config doesn't support max size policy `ENTRY_COUNT`. Please select a valid one: [PER_NODE, PER_PARTITION, USED_HEAP_PERCENTAGE, USED_HEAP_SIZE, FREE_HEAP_PERCENTAGE, FREE_HEAP_SIZE, USED_NATIVE_MEMORY_SIZE, USED_NATIVE_MEMORY_PERCENTAGE, FREE_NATIVE_MEMORY_SIZE, FREE_NATIVE_MEMORY_PERCENTAGE].
```

I suspect that, along with the [update to Spring Boot 2.4.0](https://github.com/codecentric/spring-boot-admin/commit/9fa2dfd8b22041407a5eebb02d96bf8a0270acfa#diff-b5545a4bb6bb503f72f9de2d1387a6db8200120cb6a9353c66996b0620cabe7e) the Hazelcast configuration change introduced the failure:

https://github.com/codecentric/spring-boot-admin/blob/b9206622073ac3077b8980aceabf5a485c8d01a0/spring-boot-admin-samples/spring-boot-admin-sample-hazelcast/src/main/java/de/codecentric/boot/admin/SpringBootAdminHazelcastApplication.java#L78

Setting the `MaxSizePolicy` of the `sentNotificationsMap` to `PER_NODE` for example fixes the test.

But as I have no experience with Hazelcast nor what the intention here is, I'll just leave my findings here for further analysis. :-)

--- a/spring-boot-admin-samples/spring-boot-admin-sample-hazelcast/src/main/java/de/codecentric/boot/admin/SpringBootAdminHazelcastApplication.java
+++ b/spring-boot-admin-samples/spring-boot-admin-sample-hazelcast/src/main/java/de/codecentric/boot/admin/SpringBootAdminHazelcastApplication.java
@@ -21,6 +21,7 @@ import com.hazelcast.config.EvictionConfig;
 import com.hazelcast.config.EvictionPolicy;
 import com.hazelcast.config.InMemoryFormat;
 import com.hazelcast.config.MapConfig;
+import com.hazelcast.config.MaxSizePolicy;
 import com.hazelcast.config.MergePolicyConfig;
 import com.hazelcast.config.TcpIpConfig;
 import com.hazelcast.spi.merge.PutIfAbsentMergePolicy;
@@ -75,7 +76,8 @@ public class SpringBootAdminHazelcastApplication {
 		// the same notification to be sent by multiple instances
 		MapConfig sentNotificationsMap = new MapConfig(DEFAULT_NAME_SENT_NOTIFICATIONS_MAP)
 				.setInMemoryFormat(InMemoryFormat.OBJECT).setBackupCount(1)
-				.setEvictionConfig(new EvictionConfig().setEvictionPolicy(EvictionPolicy.LRU))
+				.setEvictionConfig(new EvictionConfig().setEvictionPolicy(EvictionPolicy.LRU)
+						.setMaxSizePolicy(MaxSizePolicy.PER_NODE))
 				.setMergePolicyConfig(new MergePolicyConfig(PutIfAbsentMergePolicy.class.getName(), 100));
 
 		Config config = new Config();

